### PR TITLE
feat: add bash script to automate branch deployment and port config

### PR DESCRIPTION
**Pull request by whhygee**
Added scripts for automating branch deployments on remote servers

### CODE REVIEW

**Comment by Aksh-Bansal-dev** [comment]
Let's also provide a link to `CONTRIBUTING.md` in the README

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+    ```
+    <module_name> : <week_no> : <PR_Title>
+    ```
+
+    `<module_name> `- You only have to specify the module name in lowercase letters. For example, os-2, ac-2
+
+    ` <week_no> `- Specify the week number in the PR title that is the week in which this task has been assigned to you.
+
+    `<PR_Title> `- Specify the title of the PR starting with the lowercase letter of the first word.
```
Comment (PranshuNayak) [inline CONTRIBUTING.md:70]: We can use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0-beta.2/) for PR title

Comment (akshatnema) [reply CONTRIBUTING.md:70]: We can, but Fusion isn't released its beta version right now. So there is no need for conventional commits right now. According to Atul sir's directions, we have to show the module name along with the week no PR in the commits of the repo.

Comment (PranshuNayak) [reply CONTRIBUTING.md:70]: ok

**Pull request merged**